content/en/admin/install.md

```diff
@@ -42,7 +42,7 @@ echo "deb [signed-by=/usr/share/keyrings/postgresql.asc] http://apt.postgresql.o
 ```bash
 apt update
 apt install -y \
-  imagemagick ffmpeg libpq-dev libxml2-dev libxslt1-dev file git-core \
+  imagemagick ffmpeg libvips-tools libpq-dev libxml2-dev libxslt1-dev file git-core \
```

Comment:
Should we really have people install both here? It do not feel useful. Maybe let's keep the instructions for IM only for now, and switch them to vips for 4.4 when it becomes the default?

Reply:
So here's my rationale on this: because on Debian most of imagemagick gets installed when you install libvips, including it makes only a small impact and makes it easier to enable the option later if they want to try it and less friction when it switches to become default.